src/ragas/dataset_schema.py

```diff
@@ -24,12 +24,13 @@ class SingleTurnSample(BaseEvalSample):
     response: t.Optional[str] = None
     multi_responses: t.Optional[t.List[str]] = None
     reference: t.Optional[str] = None
-    rubric: t.Optional[t.Dict[str, str]] = None
+    rubrics: t.Optional[t.Dict[str, str]] = None
 
 
 class MultiTurnSample(BaseEvalSample):
     user_input: t.List[t.Union[HumanMessage, AIMessage, ToolMessage]]
     reference: t.Optional[str] = None
+    rubrics: t.Optional[t.Dict[str, str]] = None
```

Comment:
if something is `Optional` the default can be something like an empty dict too - doesn't have to by None always. this is a pattern we have in the codebase, something we should think about

Reply:
An argument for None is, that it is easy to check and verify if the value is None in case the metric requires a rubric to be specified.